_saas-integrations/outreach/v1/outreach-v1.md

```diff
@@ -0,0 +1,122 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Outreach
+permalink: /integrations/saas/outreach
+keywords: outreach, integration, schema, etl outreach, outreach etl, outreach schema
+layout: singer
+# input: false
+
+key: "outreach-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "outreach"
+display_name: "Outreach"
+
+singer: true
+status-url: "https://status.outreach.io/"
+
+tap-name: "Outreach"
+repo-url: https://github.com/singer-io/tap-outreach
+
+this-version: "1"
+
+api: |
+  [{{ integration.display_name}} REST API](https://api.outreach.io/api/v2/docs){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true
+
+api-type: "platform.outreach"
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: |
+      **An Outreach OAuth application**. If you have not set one up yet in {{ integration.display_name }}, refer to the [{{ integration.display_name }} API documentation](https://api.outreach.io/api/v2/docs#authentication) for more information.
```

Comment:
I don't believe this is necessary... the API docs linked here don't talk about having the user actually set up an application or anything. The user setting this up in Stitch should just have to click **Authorize** and then enter their Outreach creds to complete the setup.

That's usually the appeal of OAuth - you don't have to actually do anything.